_inc/lib/xmlrpc-async-call.php

```diff
@@ -0,0 +1,131 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+
+/**
+ * Make XMLRPC async calls to WordPress.com
+ *
+ * This class allows you to enqueue XMLRPC calls that will be grouped and sent
+ * at once in a multi-call request at shutdown.
+ *
+ * Usage:
+ *
+ * require_lib( 'xmlrpc-async-call' );
```

Comment:
Any benefit to adding this to the connection package instead? Provide a way for any connected plugin to send async calls?

Reply:
hmmm yes... I think you're right. It can be useful if properly documented ;) will add to the package docs